src/clients/c++/examples/CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ install(
 add_executable(ensemble_image_client ensemble_image_client.cc)
 target_link_libraries(
   ensemble_image_client
-  PRIVATE request_static
+  PRIVATE Trtis::request_static
```

Comment:
Can we make the namespace be TRTIS (all upper case)?

Reply:
Yes, of course; it's actually better since it's an acronym. Done

I was thinking about making it a variable, but then again might need to be higher up in the cmake hierarchy.